website/app/services/head-data.js

```diff
@@ -1,24 +1,28 @@
 import HeadDataService from 'ember-meta/services/head-data';
+import config from 'ember-get-config';
 
 export default class CustomHeadDataService extends HeadDataService {
-  // this is needed or `head-data.js` will crush
-  get config() {
-    return {};
-  }
-
   get title() {
     return (
       this.currentRouteMeta?.title ??
       this.currentRouteMeta?.frontmatter?.title ??
-      'Helios Design System'
+      config['ember-meta'].title
```

Comment:
still not a big fan of using the ember config this way (not sure what is the added value), but I can live with that :)